docs/src/tutorials/model_catalogue.qmd

```diff
@@ -51,6 +53,43 @@ end
 plot(plts...)
 ```
 
+## Importing PyTorch models
+
+The package supports generating counterfactuals for any neural network that has been previously defined and trained using PyTorch, regardless of the specific architectural details of the model. To generate counterfactuals for a PyTorch model, save the model inside a `.pt` file and call the following function:
```

Comment:
Not fully sure about this part since we're defining a loss function in the `function ∂ℓ`:

`obj_loss = torch.nn.BCEWithLogitsLoss()(output, target)`.

What do you think, should we change the documentation or tune our implementation?